ansible/roles/test/tasks/mtu.yml

```diff
@@ -37,4 +37,5 @@
     ptf_test_params:
       - testbed_type='{{testbed_type}}'
       - router_mac='{{ansible_Ethernet0['macaddress']}}'
+      - testbed_mtu=0 # 0 -- using default. The value includes the 14 bytes Layer 2 Ethernet header.
```

Comment:
Can you pass this as test parameter if you want to make it configurable. This way, it seems user has to edit this value before running the test

Reply:
Agree. Done.

Two scenarios has been tested  with the parameter of mtu  and no the parameter.
passed.